megaqc/model/models.py

```diff
@@ -141,6 +142,41 @@ def type(self):
         """
         return self.schema_json.get('type')
 
+    @hybrid_property
+    def nice_section(self):
+        """
+        A more human-readable version of the "section" field
+        """
+        return self.data_section.replace('_', ' ')
+
+    @nice_section.expression
+    def nice_section(cls):
+        # Technically string replacement isn't in the SQL standard, but it is 
+        # implemented by all the DBMSs:
+        # https://en.wikibooks.org/wiki/SQL_Dialects_Reference/Functions_and_expressions/String_functions
+        return func.replace(cls.data_section, '_', ' ')
+
+    @hybrid_property
+    def nice_key(self):
+        """
+        A more human-readable version of the "key" field
+        """
+        return self.data_key.replace('_', ' ')
+
+    @nice_key.expression
+    def nice_key(cls):
+        # Technically string replacement isn't in the SQL standard, but it is
+        # implemented by all the DBMSs:
+        # https://en.wikibooks.org/wiki/SQL_Dialects_Reference/Functions_and_expressions/String_functions
+        return func.replace(cls.data_key, '_', ' ')
+
+    @hybrid_property
+    def nice_name(self):
+        """
+        A human-readable representation of the full name of this data type
+        """
+        return self.nice_section + ': ' + self.nice_key
```

Comment:
`self.nice_key` seems to be identical to `self.nice_section` in all cases I could see. maybe just do one, or is there different attribute that should be used for `self.nice_section`?

![Screenshot from 2020-02-28 14-31-50](https://user-images.githubusercontent.com/5105646/75553311-6eb14080-5a38-11ea-96cf-f89f7ab67dd0.png)

Reply:
Well, `data_key` seems to include `data_section` as a substring in all cases. I agree it seems redundant, I'm actually keeping it just in case it breaks for some tool that doesn't work like that. I have no idea how the MultiQC report parsing works, so I'm worried about breaking it. Unless @ewels can clarify?

Reply:
If I were to write the parser from scratch, I'd want `data_section` to be `fastqc`, and `data_key` to be `gc_content`, and we'd have none of these double underscores and duplicated labels

Reply:
I suspect that this could be due to the MultiQC data structure? There we have modules (FastQC, Picard..) and each module can have sections (GC content, Insert Size..). But some modules have only one section.

I'm not sure about the specific variables though as I didn't write any of this code, so not sure whether we can remove any redundancy here.

Reply:
That sounds like exactly what's happened. I bet if we look at the parsing script, we'd find the module sections joined together by an underscore

Reply:
Aha, it is redundant by definition! https://github.com/ewels/MegaQC/blob/130cafdb1f74b042385d0645a7acf11e8129f8d7/megaqc/api/utils.py#L140-L143. So I can remove the section label entirely.